modules/javafx.base/src/test/java/test/com/sun/javafx/binding/SelectBindingTest.java

```diff
@@ -102,13 +107,14 @@ public void removePropertyChangeListener(String property, PropertyChangeListener
     private ObservableList<?> dependencies;
 
     @BeforeAll
-    public static void setUpClass() {
-        System.err.println("SelectBindingTest : log messages are expected from these tests.");
+    public static void setUpClass() throws Exception {
```

Comment:
Minor: this method previously didn't have `throws Exception` and probably doesn't need it.

Reply:
done